### PR TITLE
STR-53 -- FE: Create sass variables for our color pallet in our application

### PR DESCRIPTION
**Pull request by frlzjosh**
# Description
'Write a meaningful description of your task'
  Create sass variables for our color pallet in our application
# Testing
'What does the reviewer need to do to test your PR?'
 1. Before switching to my branch, keep an open version of our project in the browser. Then checkout to 
  STR-53 to assure that all the colors are still the same
 2. Checkout my files changed and the naming of my variables, give any critique needed
# Installation
'Are there any dependencies the reviewer needs to download?'

### CODE REVIEW

File: resources/sass/global/_variables.scss [hunk 1]
```diff
@@ -24,7 +22,40 @@ $orange: #f6993f;
...
 $green: #38c172;
 $teal: #4dc0b5;
-$cyan: #6cb2eb;
-$i-white: #FFFFFF;
-$i-blue: #2699FB;
-$primary: $i-blue;
+
+$primaryColor: rgb(38,153,251);
+$primaryColor--light: lighten($primaryColor, 0.6);
+$primaryColor--med: rgb(16, 109, 188);
```
Comment (BrianLinggadjaja) [inline resources/sass/global/_variables.scss]: I think --med should be expanded to medium or use another name for it.

File: resources/sass/global/_variables.scss [hunk 2]
```diff
@@ -24,7 +22,40 @@ $orange: #f6993f;
...
+$primaryColor--dark: darken($primaryColor, 0.8);
+$primaryColor--shadow: rgba(38, 153, 251, 0.2);
+
+$secondaryColor: #FFFFFF;
+$secondaryColor--light: lighten($secondaryColor, 0.6);
+$secondaryColor--dark: darken($secondaryColor, 0.8);
+
+// Shades (Standard Shades)
+$white: rgb(245, 245, 245);
+$white--light: rgb(255,255,255);
```
Comment (BrianLinggadjaja) [inline resources/sass/global/_variables.scss]: One last little small thing, is to make sure there are spaces after each comma.

**Review by BrianLinggadjaja — CHANGES_REQUESTED** [review]

File: resources/sass/global/_variables.scss (hunk 1 above)
Comment (frlzjosh) [reply resources/sass/global/_variables.scss]: coo

File: resources/sass/global/_variables.scss (hunk 2 above)
Comment (frlzjosh) [reply resources/sass/global/_variables.scss]: ayy

**Review by ekno27 — APPROVED** [review]

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
As we discussed, let's keep naming consistent, like:
--lighter
--light
--medium
--dark
--darker

**Review by BrianLinggadjaja — APPROVED** [review]

**Review by andrewMETALAB — APPROVED** [review]
Cool 👍

**Pull request merged**